src/SWP/Bundle/AnalyticsBundle/Resources/config/doctrine-orm/ArticleStatistics.orm.yml

```diff
@@ -13,6 +13,9 @@ SWP\Bundle\AnalyticsBundle\Model\ArticleStatistics:
         pageViewsNumber:
             type: integer
             nullable: true
+        internalClickRate:
```

Comment:
why not name it just clickRate? what's the difference between internal click rate and click rate?

Reply:
it's about click rate but calculated from internal type of page views